positron/comms/data_explorer-backend-openrpc.json

```diff
@@ -265,9 +265,17 @@
 		},
 		{
 			"name": "get_column_profiles",
-			"summary": "Request a batch of column profiles",
-			"description": "Requests a statistical summary or data profile for batch of columns",
+			"summary": "Async request a batch of column profiles",
+			"description": "Async request for a statistical summary or data profile for batch of columns",
 			"params": [
+				{
+					"name": "callback_id",
```

Comment:
It makes me a little itchy to have to complicate the contract b/c of the limitations of the backend, but I don't see a better way than what you've done here.